### ISSUE TITLE
Emoji support in notebook titles Android

### ISSUE THREAD

**Issue by JaneX8** [laurent22/joplin#8811]
First ones are setup on Desktop using "Select Emoji" option.

Last ones are simply an emoji as typed on my mobile keyboard in the title text field. It would be nice if emojis are supported on Android the same way as on Desktop.

![Screenshot_2023-09-10-12-41-44-07_97a2b898327add53e982bbebadc3de52.jpg](https://github.com/laurent22/joplin/assets/5116641/4bb716e7-0747-461d-8207-123af7d42308)

**Comment by laurent22** [comment]
Both Emojis and custom icons work for me. Which version is it?

**Comment by JaneX8** [comment]
Version 2.12.2. But there is no "select emoji" button (like in Desktop) on the mobile version, when creating or editing notebooks. 

On Android I can just insert a custom emoji using an emoji keyboard but it will be part of the title (text), as shown in the last three lines my screenshot.

**Comment by laurent22** [comment]
Right, I see what you mean. Indeed currently we have no UI for this, but as a workaround you can set the icon on desktop, and after sync they will show up on mobile too.

**Comment by JaneX8** [comment]
I'm doing that, it's no big deal but I figured it's worth reporting as a potential minor UI improvement.

_Overall, all works great. I found Joplin by recommendation of GPT4 and love it. It's been my daily information (personal knowledge) management system utilising the PARA-method for a few weeks now with Dropbox E2EE sync and it works absolutely fantastic. Great job to all developers working on it 👌🏻._
